sunscreen_runtime/src/run.rs

```diff
@@ -157,25 +184,70 @@ pub unsafe fn run_program_unchecked<E: Evaluator + Sync + Send>(
         data.push(AtomicCell::new(None));
     }
 
+    #[cfg(feature = "debugger")]
```

Comment:
Don't need to change for this code review, but check how Brian and I make a session for ZKP. It's way cleaner.